### PR TITLE
add more image tests for go bindings

### PR DESCRIPTION
**Pull request by baude**
adding more image tests for go bindings.  one big change is that the params were converted from map[string]string to url.values to account for the ability to send []string as query params

Signed-off-by: Brent Baude <bbaude@redhat.com>

### CODE REVIEW

File: pkg/bindings/containers/containers.go [hunk 1]
```diff
@@ -21,28 +22,28 @@ func List(ctx context.Context, filters map[string][]string, all *bool, last *int
 		return nil, err
 	}
 	var containers []lpapiv2.ListContainer
-	params := make(map[string]string)
+	params := url.Values{}
 	if all != nil {
-		params["all"] = strconv.FormatBool(*all)
+		params["all"] = []string{strconv.FormatBool(*all)}
```
Comment (jwhonce) [inline pkg/bindings/containers/containers.go]: params.Set("all", strconv.FormatBool(*all)) should be used.  Same below.

File: pkg/bindings/images/images.go [hunk 2]
```diff
@@ -33,16 +35,16 @@ func List(ctx context.Context, all *bool, filters map[string][]string) ([]*handl
 	if err != nil {
 		return nil, err
 	}
-	params := make(map[string]string)
+	params := url.Values{}
 	if all != nil {
-		params["all"] = strconv.FormatBool(*all)
+		params["all"] = []string{strconv.FormatBool(*all)}
```
Comment (jwhonce) [inline pkg/bindings/images/images.go]: Should be params.Set("all", strconv.FormatBool(*all)).  Same below.

File: pkg/bindings/images/images.go [hunk 3]
```diff
@@ -185,3 +193,35 @@ func Tag(ctx context.Context, nameOrID, tag, repo string) error {
...
+	conn, err := bindings.GetClient(ctx)
+	if err != nil {
+		return "", err
+	}
+	params := url.Values{}
+	for _, change := range changes {
+		params.Add("changes", change)
+	}
+	if message != nil {
+		params["message"] = []string{*message}
```
Comment (jwhonce) [inline pkg/bindings/images/images.go]: params.Set("message", *message)

File: pkg/bindings/test/common_test.go [hunk 4]
```diff
@@ -20,9 +20,18 @@ type testImage struct {
...
+	devPodmanBinaryLocation     string = "../../../bin/podman"
 	defaultPodmanBinaryLocation string = "/usr/bin/podman"
 )
 
+func getPodmanBinary() string {
+	_, err := os.Stat(devPodmanBinaryLocation)
+	if os.IsNotExist(err) {
+		return defaultPodmanBinaryLocation
+	}
+	return devPodmanBinaryLocation
```
Comment (jwhonce) [inline pkg/bindings/test/common_test.go]: On permission denied errors, you are going to return devPodmanBinaryLocation

File: pkg/bindings/volumes/volumes.go [hunk 5]
```diff
@@ -73,9 +74,9 @@ func Remove(ctx context.Context, nameOrID string, force *bool) error {
 	if err != nil {
 		return err
 	}
-	params := make(map[string]string)
+	params := url.Values{}
 	if force != nil {
-		params["force"] = strconv.FormatBool(*force)
+		params["force"] = []string{strconv.FormatBool(*force)}
```
Comment (jwhonce) [inline pkg/bindings/volumes/volumes.go]: params.Set("force", strconv.FormatBool(*force))

File: pkg/bindings/test/common_test.go (hunk 4 above)
Comment (baude) [reply pkg/bindings/test/common_test.go]: i did that on purpose ... thats ok to me

**Comment by jwhonce** [comment]
LGTM

**Pull request merged**